homework/index.js

```diff
@@ -29,19 +29,149 @@
     });
     return elem;
   }
+  // ==========start===========
+  /* cSpell: disable */
+  const repoInfo = (index, repoName) => {
```

Comment:
You could send the repo data instead of passing both array and index.

Reply:
I tried to do so but I got 'Network request failed'